RedJanvier/eshuri-bank-be
# Account Cards CRUD

#### What does this PR do?
Have Rest endpoints on account cards

#### Description of Task to be completed?
Have the following endpoints working

POST `/api/cards`
GET `/api/cards`
GET `/api/cards/<card-id>`
DELETE `/api/cards/<card-id>`
PUT `/api/cards/<card-number>/change-pin`
PUT `/api/cards/<card-number>/pin-reset`


#### How should this be manually tested
After cloning you can use postman to check if this routes are working...
POST `/api/cards` it has no body for now. Everything is dynamically generated on the backend (sensitive informaion like account card number, cvv and pin)

PUT `/api/cards/<card-number>/change-pin`
`Content-Type: application/json
{
  "oldPin": "",
  "newPin": ""
}`

PUT `/api/cards/<card-number>/pin-reset`
`Content-Type: application/json
{
  "newPin": "1234"
}`
